File: QViz.NET/Periscope.QViz/JSON/Module.cs
using System;

namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for Project Module Details
	/// </summary>
	public class Module
	{
		/// <summary>
		/// Identifier of the Project Module
		/// </summary>
		public string moduleId;
		
		/// <summary>
		/// Name of the Project Module
		/// </summary>
		public string name;
		
		/// <summary>
		/// Is this Project Module Active?
		/// </summary>
		public bool isActive;
		
		/// <summary>
		/// Date and Time on which this Model was Created
		/// </summary>
		public DateTime createdOn;
		
		/// <summary>
		/// Date and Time on which this Model was last Updated
		/// </summary>
		public DateTime updatedOn;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public Module()
		{
			this.moduleId = "";
			this.name = "";
			this.isActive = false;
			this.createdOn = DateTime.Now;
			this.updatedOn = DateTime.Now;
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/Module.java
package com.periscope.qviz.json;

import java.util.Date;

/**
 * JSON Model for Project Module Details
 */
public class Module {

	/**
	 * Identifier of the Project Module
	 */
	public String moduleId;

	/**
	 * Name of the Project Module
	 */
	public String name;

	/**
	 * Is this Project Module Active?
	 */
	public Boolean isActive;

	/**
	 * Date and Time on which this Model was Created
	 */
	public Date createdOn;

	/**
	 * Date and Time on which this Model was last Updated
	 */
	public Date updatedOn;

	/**
	 * Default Constructor
	 */
	public Module() {
		this.moduleId = "";
		this.name = "";
		this.isActive = false;
		this.createdOn = null;
		this.updatedOn = null;
	}

}
